proxy/Dockerfile

```diff
@@ -10,6 +10,7 @@ ENV NGINX_ENV=local \
     TZ=UTC
 
 RUN apk add --no-cache 'ruby=~2.7' \
+ && apk --no-cache upgrade \
```

Comment:
Improvement suggestion: As we use a Docker layer cache in CI, we could preemptively create a cache busting layer like: `ARG CACHE_BUST=2021-06-07` that can then just be updated if we need to explicitly do a similar forced update.